src/MongoDB.Analyzer/Core/Poco/PocoDataFiller.cs

```diff
@@ -65,13 +73,48 @@ private static object HandleClass(Type memberType, int levelsLeft)
     private static Array HandleSingleDimensionalArray(Type arrayType) =>
         Array.CreateInstance(arrayType.GetElementType(), 0);
 
+    private static object HandleString(string memberName)
+    {
+        try
+        {
+            var caseInsensitiveMember = memberName.ToLower();
+            foreach (var jObject in s_jsonData)
+            {
+                if (jObject.First is JProperty jProperty &&
+                    caseInsensitiveMember.Contains(jProperty.Name.ToLower()))
+                {
+                    var values = ((JArray)jProperty.Value).ToObject<string[]>();
+                    return values[memberName.Length % values.Length];
+                }
+            }
+        }
+        catch
```

Comment:
Probably no need for general catch here, we are catching on an higher level.

Reply:
Done